nucliadb/nucliadb/ingest/metrics/service.py

```diff
@@ -29,6 +31,10 @@
     PROMETHEUS = False
 
 
+# Do not flood stdout with /metrics access logs
+logging.getLogger("aiohttp.access").setLevel(logging.WARNING)
```

Comment:
There are other endpoint access that will be hidden by this change on the log level?

Reply:
AFAIK Ingest does not expose any http service other than the used for metrics collection, so we are safe.